src/index.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line no-unused-vars
+import _ from 'lodash';
+import './style.css';
+
+const todoLists = [
+  {
+    description: 'Working on javascript todo app',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Work on ruby on rails',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'Work on php laravel',
+    completed: true,
+    index: 8,
+  },
+  {
+    description: 'Work on java',
+    completed: false,
+    index: 6,
+  },
+];
+
+todoLists.forEach(todo => {
+  const li = document.createElement('li');
+  const ul = document.getElementById('todo-lists');
+  li.innerText = todo.description;
+  ul.appendChild(li);
+});
```

Comment:
- [ ] Optional

- To met the minimalistic requirement of design we can also add a button and a checkbox at the start of every list item. This will help you in the later part of the milestone.

![image](https://user-images.githubusercontent.com/53019916/126498972-49179b69-afe3-46a8-959d-c698f52ce0a2.png)